tests/render/test_column_section_renderer.py

```diff
@@ -1194,6 +1221,23 @@ def test_ExpectationSuiteColumnSectionRenderer_expectation_with_single_string_me
     assert result_json == expected_result_json
 
 
+@pytest.mark.unit
+def test_ExpectationSuiteColumnSectionRenderer_render_expectation_with_description(
```

Comment:
I believe I've encapsulated the primary usecases that would hit this logic within these tests but I'm not 100% confident - some manual testing seems to confirm these beliefs but let me know if you think I'm missing something.

Reply:
I think the last assert that the result is rendered as the description string is the main use case we want to test.